ras_party/controllers/pending_survey_controller.py

```diff
@@ -64,91 +67,110 @@ def pending_share_create(business_id, survey_id, email_address, shared_by, batch
     :param session: db session
     :param batch_number: batch_number
     :type batch_number: uuid
+    :param is_transfer: True if the record is for transfer survey
```

Comment:
```suggestion
    :param is_transfer: if the request is to transfer surveys
```

(may as well keep these consistent given they're the same variable)

Reply:
👍